monkey/monkey_island/cc/services/telemetry/processing/system_info_collectors/system_info_telemetry_dispatcher.py

```diff
@@ -3,12 +3,10 @@
 
 from common.common_consts.system_info_collectors_names import (AWS_COLLECTOR, ENVIRONMENT_COLLECTOR, HOSTNAME_COLLECTOR,
                                                                PROCESS_LIST_COLLECTOR, SCOUTSUITE_COLLECTOR)
-from monkey_island.cc.services.telemetry.processing.system_info_collectors.aws import process_aws_telemetry
-from monkey_island.cc.services.telemetry.processing.system_info_collectors.environment import \
-    process_environment_telemetry
-from monkey_island.cc.services.telemetry.processing.system_info_collectors.hostname import process_hostname_telemetry
-from monkey_island.cc.services.telemetry.processing.system_info_collectors.scoutsuite import \
-    process_scout_suite_telemetry
+from .aws import process_aws_telemetry
```

Comment:
Why are you transitioning to relative imports? In python 2 it was definitely a bad idea.
In python 3 it's controversial. We should have a consistent decision for these all over monkey code.
The decision used to be explicit imports from project's root, if we change it, let's have @mssalvatore involved

Reply:
Because if this was an absolute import it took 2 lines. It's not only more readable, but also doesn't freak out our import formatter.

Reply:
I understand, but I'd rather we stay consistent. Whether we decide to use relative or absolute imports.
I'll approve these changes here, but you should discuss this with @mssalvatore and formalize this decision

Reply:
I vote for absolute, but I'm open to a debate.

Reply:
If we convert to "relative package scope imports, everything else - absolute", a lot of absolute imports in file could hint that a file should be moved somewhere + imports are more readable. For now I'll revert, but we need **some** solution to long (and growing) absolute imports. One solution could be to change python paths to omit `monkey_island.cc` from import paths, because it's redundant.

Reply:
I'm for absolute imports all the way, but I understand @VakarisZ 's point.
The ideal solution the way I see it, is to explicitly define our "framework stack", and the implied dependencies (which have to be acyclic).
Meaning something like: "Layer 1: basic utils & framework", "Layer 2: data model", "Layer 3: independent services", "Layer 4: resources", "Layer 5: application/entrypoint".
We can choose a different model (like merging the data & services layers), but choosing one and sticking to it is key.

Once we do that, each module in each layer (e.g. a specific service) can be built as an independent package relying only on packages in lower layers. We can still keep all packages in the same monkey repo though.

Having these modules defined as independent packages has a lot of advantages, one of them being short absolute paths (since their root is the specific sub-package)